torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1113,7 +1125,7 @@ int listSelect<Shared<BoolList>>(Stack& stack) {
   pop(stack, list, idx);
 
   auto element = getBoolItem(list->elements(), idx);
-  push(stack, std::move(element));
+  push(stack, element);
```

Comment:
Why did we loose the move here? Seems like a regression?

Reply:
Hmmm iirc this is done by clang-tidy hook. It was suggesting moving a bool doesn't make a perf difference and I thought that makes sense. I could bring it back if this might cause regression.

Reply:
Ohhh I haven't noticed it's a bool. Yeah, in that case it shouldn't be there